docs/iaas.md

```diff
@@ -98,3 +102,68 @@ http://localhost:30000/d/openstack-overview
 - Use the following credentials:
   - username: `admin`
   - password: `pass`
+
+### Ceph
+
+The SCS IaaS reference implementation (OSISM) currently supports two methods for deploying Ceph:
+- [ceph-ansible](https://github.com/ceph/ceph-ansible)
+- [rook operator](https://github.com/rook/rook)
```

Comment:
this is not yet supported; I don't know the exact timeline when rook will be supported:

https://osism.tech/docs/guides/deploy-guide/services/ceph